test/views/declaration/summary/UnionAndNationalCodesViewSpec.scala

```diff
@@ -76,9 +76,9 @@ class UnionAndNationalCodesViewSpec extends UnitViewSpec with ExportsTestData wi
 
     "display additional information without change buttons" when {
 
-      "mode is Print" in {
+      "actionsEnabled is false" in {
```

Comment:
This case is already tested staring from line 106. So you can remove this block.

Reply:
Good spots!  I just focused on fixing the compile issues!